sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -495,8 +495,12 @@ group varying "Varying linkage"
 
 		# different interpolation
 		case differing_interpolation_2
+			# This deviates from C++ dEQP here. GLSL ES 3.00.4 specifies interpolation
+			# qualifier of variables with the same name declared in all linked shaders
+			# must match, in Section 4.3.9 Interpolation.
 			version 300 es
 			desc "varying interpolation different (smooth vs. centroid)"
+			expect link_fail
```

Comment:
I think this shouldn't fail because centroid is an auxiliary interpolation qualifier. See https://cs.chromium.org/chromium/src/third_party/angle/src/compiler/translator/ShaderVars.cpp?q=ShaderVars&sq=package:chromium&l=26